Assets/MixedRealityToolkit.Examples/Demos/Gltf/Models/Duck/README.md

```diff
@@ -0,0 +1,14 @@
+# Duck
+## Screenshot
+
+![screenshot](screenshot.png)
+
+## License Information
+
+Copyright 2006 Sony Computer Entertainment Inc.
```

Comment:
before merging this, please get approval internally and register the usage (i can help)

Reply:
The duck makes it's triumphant return! https://www.youtube.com/watch?v=7UaDvIPpQnQ

Reply:
"Retro" is in these days :P

Reply:
@davidkline-ms I switched out the duck model for a microsoft avocado model. 

@Cameron-Micka sorry, the duck won't be making a triumphant return :(